distributor-node/src/services/networking/query-node/queries/queries.graphql

```diff
@@ -107,7 +107,17 @@ fragment StorageBucketOperatorFields on StorageBucket {
 }
 
 query getActiveStorageBucketOperatorsData {
-  storageBuckets(where: { operatorStatus: { isTypeOf_eq: "StorageBucketOperatorStatusActive" } }) {
+  storageBuckets(
+    where: {
+      operatorStatus: { isTypeOf_eq: "StorageBucketOperatorStatusActive" }
+      operatorMetadata: {
+        OR: [
+          { nodeOperationalStatus_isNull: true }
+          { nodeOperationalStatus: { isTypeOf_eq: "NodeOperationalStatusNormal" } }
```

Comment:
This filter would skip any nodes with future maintenance as well. We probably should get all the operators and later only filter those that are under maintenance **currently**

Reply:
Done in [0537ee7](https://github.com/Joystream/joystream/pull/4793/commits/0537ee74c8f5caa8a14217668fc7cab59a5ca68c)